src/Controls/src/Core/BindableObject.cs

```diff
@@ -11,18 +11,24 @@
 
 namespace Microsoft.Maui.Controls
 {
-	/// <include file="../../docs/Microsoft.Maui.Controls/BindableObject.xml" path="Type[@FullName='Microsoft.Maui.Controls.BindableObject']/Docs/*" />
+	/// <summary>
+	/// Provides a mechanism by which application developers can propagate changes that are made to data in one object to another, by enabling validation, type coercion, and an event system.
+	/// </summary>
+	/// <remarks>The <see cref="BindableObject" /> class provides a data storage mechanism that enables the application developer to synchronize data between objects in response to changes, for example, between the View and View Model in the MVVM design pattern. All of the visual elements in the <c>Microsoft.Maui.Controls</c> namespace inherit from <see cref="BindableObject" /> class, so they can all be used to bind the data behind their user interface elements to View Models that are supplied by the application developer.</remarks>
 	public abstract class BindableObject : INotifyPropertyChanged, IDynamicResourceHandler
 	{
 		IDispatcher _dispatcher;
 
-		// return the dispatcher that was available when this was created,
-		// otherwise try to find the nearest dispatcher (probably the window/app)
-		/// <include file="../../docs/Microsoft.Maui.Controls/BindableObject.xml" path="//Member[@MemberName='Dispatcher']/Docs/*" />
+		/// <summary>
+		/// Gets the dispatcher that was available when this was created,
+		/// otherwise tries to find the nearest available dispatcher (probably the Window/App one).
```

Comment:
There's a sort of ambiguous pronoun here with the use of "this". Also, shouldn't Window/App not be capitalized? For a second it referred to Windows  (the OS) because of it

```suggestion
		/// Gets the dispatcher that was available when the <see cref="BindableObject"/>was created,
		/// otherwise tries to find the nearest available dispatcher (probably the window's/app's).
```